BrainCloudClient/src/BrainCloud/Client/BrainCloud/BrainCloudWrapper.cs

```diff
@@ -129,11 +163,16 @@ public static BrainCloudWrapper GetInstance()
             if (_instance == null)
             {
                 GameObject go = new GameObject(GAMEOBJECT_BRAINCLOUD);
+                
                 _instance = go.AddComponent<BrainCloudWrapper>();
+#pragma warning disable 618
+                _instance.client = BrainCloudClient.Get();

```

Comment:
Hmmm shouldn't this end up being new BrainCloudClient() ?? not a public get() ? i'll continue reviewing